src/sentry/static/sentry/app/components/charts/releaseSeries.tsx

```diff
@@ -31,33 +55,61 @@ async function getOrganizationReleases(api, organization, conditions = null) {
     }
   });
   api.clear();
-  return api.requestPromise(`/organizations/${organization.slug}/releases/stats/`, {
+  return (api.requestPromise(`/organizations/${organization.slug}/releases/stats/`, {
     includeAllArgs: true,
     method: 'GET',
     query,
-  });
+  }) as unknown) as [ReleaseMetaBasic[], any, JQueryXHR];
 }
 
-class ReleaseSeries extends React.Component {
-  static propTypes = {
+type Props = WithRouterProps & {
+  api: Client;
+  organization: Organization;
+  children: (s: State) => React.ReactNode;
+  projects: Readonly<number[]>;
+  environments: Readonly<string[]>;
+  start: DateString;
+  end: DateString;
+  period?: string;
+  utc?: boolean | null;
+  releases?: ReleaseMetaBasic[] | null;
+  tooltip?: EChartOption.Tooltip;
+  memoized?: boolean;
+  preserveQueryParams?: boolean;
+  emphasizeReleases?: string[];
+  queryExtra?: Query;
+};
+
+type State = {
+  releases: ReleaseMetaBasic[] | null;
+  releaseSeries: Series[];
+};
+
+class ReleaseSeries extends React.Component<Props, State> {
+  static propTypes: any = {
```

Comment:
You can likely delete the propTypes. They sometimes interfere with typescript, and can be tricky to get aligned.